PORTING.md

```diff
@@ -0,0 +1,43 @@
+# prplMesh porting guide
+
+## Prerequisites and dependencies
+
+* Toolchain:
+    * Need to support C++11.
+    * Advisable to use gcc version 5.* and onwards.
+    * Ideally use the cross compilation toolchain already existing for your target device (use the same libc).
+        * If not possible due to the mentioned requirements, it is possible to use compilation toolchains in Debian, Ubuntu or generate your own one (Buildroot).
+        * In order to avoid using two different libc installations you can consider building prplMesh statically. (Would need to link against static libraries).
+
+* Dependencies:
+    * Basic ones are covered in the main README file. (Those should cover any of the BWLs supported).
```

Comment:
I think in the main README we should at least remove clang-format and anything else that's not strictly needed for prplMesh itself.
We can keep a second list for all the tools/scripts and so on that are useful but not mandatory.

(not for this PR)